libs/shared/src/consts/notification-item-buttons/notificationItemButton.ts

```diff
@@ -33,21 +33,21 @@ export const darkButtonStyle: IStyleButtons = {
     backGroundColor: 'linear-gradient(99deg,#DD2476 0% 0%, #FF512F 100% 100%)',
     fontColor: '#FFFFFF',
     removeCircleColor: 'white',
-    fontFamily: 'Lato',
+    fontFamily: 'inherit',
   },
-  secondary: { backGroundColor: '#3D3D4D', fontColor: '#FFFFFF', removeCircleColor: '#525266', fontFamily: 'Lato' },
-  clicked: { backGroundColor: 'white', fontColor: '#FFFFFF', removeCircleColor: '#525266', fontFamily: 'Lato' },
+  secondary: { backGroundColor: '#3D3D4D', fontColor: '#FFFFFF', removeCircleColor: '#525266', fontFamily: 'inherit' },
+  clicked: { backGroundColor: 'white', fontColor: '#FFFFFF', removeCircleColor: '#525266', fontFamily: 'inherit' },
 };
 
 export const lightButtonStyle: IStyleButtons = {
   primary: {
     backGroundColor: 'linear-gradient(99deg,#DD2476 0% 0%, #FF512F 100% 100%)',
     fontColor: '#FFFFFF',
     removeCircleColor: 'white',
-    fontFamily: 'Lato',
+    fontFamily: 'inherit',
```

Comment:
font-family is inherited css prop, I don't think that you have to specify everywhere `inherit`.

Reply:
removed them and made the `fontFamily`optional type